cosi-driver/cmd/cosi-driver/main.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"google.golang.org/grpc"
+	"k8s.io/klog"
+
+	"github.com/container-object-storage-interface/cosi-provisioner-sidecar/cosi-driver/pkg/server"
+	"github.com/container-object-storage-interface/spec/lib/go/cosi"
+)
+
+// Command line flags
+var (
+	cosiAddress = flag.String("cosi-address", "tcp://0.0.0.0:9000", "Path of the COSI driver socket that the provisioner  will connect to.")
+	showVersion = flag.Bool("version", false, "Show version.")
+	version     = "unknown"
+	// List of supported versions
+	supportedVersions = []string{"1.0.0"}
```

Comment:
I feel like this should be 0.0.1 at this point. :)

Reply:
done.